packages/tocco-util/src/units/addSystemUnit.spec.js

```diff
@@ -0,0 +1,33 @@
+import {addSystemUnit} from './addSystemUnit'
+
+describe('tocco-util', () => {
+  describe('units', () => {
+    describe('addSystemUnit', () => {
+      test('shorten numbers correctly', () => {
+        const scenarios = [
+          [1, '1'],
+          [12, '12'],
+          [123, '123'],
+          [1234, '1k'],
+          [12345, '12k'],
+          [123456, '123k'],
+          [1234567, '1M'],
```

Comment:
I would like an edge case test. e.g. 1999999 should get 1.9M or maybe 2M but not 1M

Reply:
- Unwanted outputs like 1000k is not possible anymore. 999.5K is rounded up to 1M. 
- Complex math were removed (log10).
- Needed loops were reduced by reversing array ranges by the assumption most numbers are less than 1'000'000.

Reply:
Edge Case 0 does cause an error.